lib/src/core/renovation.dart

```diff
@@ -2,6 +2,8 @@ import 'dart:core';
 
 import 'package:logger/logger.dart';
 import 'package:pedantic/pedantic.dart';
+import 'package:renovation_core/core.dart';
+import 'package:renovation_core/src/core/errors.dart';
```

Comment:
@assemmarwan maybe export the `errors.dart` within `core.dart`, what do you think?

Reply:
Actually I should import using relative reference. Good catch 👍